### PR TITLE
Motivware review blog

### PR DESCRIPTION
**Pull request by DamonClark**
For Issue #57  let me know your opinions.

### CODE REVIEW

File: content/blog/2019-09-15-the-motivware-project/index.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+title: 'Motivware Review'
+author: dclark
+date: '2019-09-15'
+image: ./hero.jpg
+tags:
+  - Project Unicorn
+  - Retrospective
+---
+
+Motivware which can be viewed [here](https://www.motivware.com), was originally just used for me to learn and improve my programming skills. Eventually I reached a point where I thought I should try converting it into an independent project. I had started to implement the core features for many software as a service applications. Which included authentication, subscription services and a multi-tenant infrastructure. I was able to string everything together by using stackoverflow and thanks to various online tutorials. The application was in a working state with many core features in place. But I started to grow tired of always coding solo and that is where Project Unicorn provided so much help! I was able to be apart of developers community and work with a remote team. Provided below I want to outline several key points that I have learned along the way.
```
Comment (rmjordas) [inline content/blog/2019-09-15-the-motivware-project/index.md]: - stackoverflow -> Stack Overflow

File: content/blog/2019-09-15-the-motivware-project/index.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+## Validate ideas sooner than later
+
+I know we are all anxious to start developing code but its important to validate your ideas too. You can start developing your new app but don't forget to ask others for their opinions. Even if its just friends or family, any feedback is better than none. The sooner you can get others opinions, the better off you will be. Maybe even think about quickly spinning up a landing page with a mailing list to evaluate demand for your app. Time is valuable and you want to develop iteratively with a customer first agile mentality.
+
+## Enjoy the challenge
+
+It can be a struggle to stay motivated and continue the process when learning software development. I have personally went through many gaps while progressing and also dealt with many challenges. It's important to stay positive and try not to get frusterated, this will just slow you down. Don't hesitate to take breaks when needed but make sure to keep a good quality habit of coding. Most importantly don't get discouraged, try to enjoy yourself and let's all have fun while we progress.
```
Comment (rmjordas) [inline content/blog/2019-09-15-the-motivware-project/index.md]: - frusterated -> frustrated

File: content/blog/2019-09-15-the-motivware-project/index.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+## Collaborate with others
+
+As a developer don't ever hesitate to reach out to the online community for help or with any questions. Coding is supposed to be a collaborative process and its important not to shy away from learning with others. Project Unicorn is designed to provide a community of developers that can learn and grow together. Before joining I was primarily learning independently and wish I would have found other remote developers sooner. I could have improved my programming skills at a much faster rate and had more fun in the process.
```
Comment (rmjordas) [inline content/blog/2019-09-15-the-motivware-project/index.md]: - Before **joining I** was primarily -> Before **joining, I** was primarily

File: content/blog/2019-09-15-the-motivware-project/index.md (hunk 1 above)
Comment (aaron2212) [inline content/blog/2019-09-15-the-motivware-project/index.md]: `apart of developers community` -> `a part of a community of developers`

Comment (aaron2212) [reply content/blog/2019-09-15-the-motivware-project/index.md]: `Provided below I want to outline...` -> `Below are several key points I want to outline that I have learned along the way`

File: content/blog/2019-09-15-the-motivware-project/index.md (hunk 3 above)
Comment (aaron2212) [inline content/blog/2019-09-15-the-motivware-project/index.md]: `process and its important` -> `process and it's important`

File: content/blog/2019-09-15-the-motivware-project/index.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+## Validate ideas sooner than later
+
+I know we are all anxious to start developing code but its important to validate your ideas too. You can start developing your new app but don't forget to ask others for their opinions. Even if its just friends or family, any feedback is better than none. The sooner you can get others opinions, the better off you will be. Maybe even think about quickly spinning up a landing page with a mailing list to evaluate demand for your app. Time is valuable and you want to develop iteratively with a customer first agile mentality.
```
Comment (aaron2212) [inline content/blog/2019-09-15-the-motivware-project/index.md]: `but its important to validate` -> `but it's important to validate`

Comment (aaron2212) [reply content/blog/2019-09-15-the-motivware-project/index.md]: `Even if its just` -> `Even if it's just`

Comment (aaron2212) [reply content/blog/2019-09-15-the-motivware-project/index.md]: `can get others opinions` -> `can get others' opinions`

File: content/blog/2019-09-15-the-motivware-project/index.md (hunk 2 above)
Comment (aaron2212) [inline content/blog/2019-09-15-the-motivware-project/index.md]: `I have personally went through` -> `I have personally gone`

**Comment by roymoran** [comment]
Don't have many comments to add that hasn't already been included by Aaron or Ruj. 

Awesome post man!

**Pull request merged**